_includes/title.html

```diff
@@ -19,10 +19,14 @@
     <h1>
         {{ page.title | xml_escape }}
         <small>{{ page.subtitle }}</small>
+        
         {% if edit_path %}
             <a class="btn btn-primary pull-right" href="{{ edit_path }}">Edit on Github</a>
         {% elsif show_news_link %}
             <a class="btn btn-primary pull-right" href="/news/atom.xml">Subscribe via Atom</a>
         {% endif %}
     </h1>
+    {% if page.author %}
```

Comment:
Also: there is no separate tutorial layout. So it's hard to restrict this to the tutorial pages. The {% if %} statement at least makes sure there is no ghost <p> when the page has no author property. Github cards would be much better btw! Maybe they can be inserted as a block, like a toc?